static/sidebar.less

```diff
@@ -62,128 +63,145 @@ img.social-icon-small{
 						}
-						&.expanded {
-							> a {
-								padding-bottom: @gutter/2;
-								margin-left: -1px;
+						&.can-infrastructure span + ul {
+							margin-top: unset;
+						}
+						span {
+							font-size: 12px;
+							letter-spacing: 1px;
+							font-weight: bold;
+							text-transform: uppercase;
+							& + [aria-pressed="true"] + ul {
+								margin-top: -10px;
 							}
 						}
-						//Third level
-						> ul {
-							border-left: 1px solid @border-color;
-							li {
-								ul {
-									margin-top: @gutter/3;
+						a {
+							line-height: @list-line-height;
+							& + ul {
+								padding-left: 5px;
+							}
+						}
+						button {
+							background: transparent;
+							border: 0;
+							color: @link-color;
+							cursor: pointer;
+							padding: 0;
+							position: absolute;
+							right: 0 - @gutter;
+							top: -@gutter/3;
+							span {
+								display: inline-block;
+								font-family: monospace;// makes the + / - buttons the same width
+								font-size: 150%;
+								padding: (@gutter * .25) @gutter (@gutter * .6);
+							}
+							&:focus, span:focus {
+								outline: 0;
+							}
+							&:focus span {
+								border-radius: 5px;
+								box-shadow: 0 0 5px @link-color;
+							}
+						}
+						ul li ul li {
+							// third forth
+							padding-left: 0;
+							span {
+								font-weight: normal;
+								display: inline-flex;
+								text-transform: none;
+								font-weight: bold;
+							}
+							ul {
+								margin-bottom: 5px;
+								li {
+									// fifth
+									line-height: @list-line-height;
+									padding-left: 0;
+									span {
+										font-weight: normal;
+									}
+									ul {
+										// sixth
+										li {
+											padding-left: 5px;
+											ul {
+												// seventh
+												li {
+													padding-left: 0;
+												}
+											}
+										}
+									}
+								}
+							}
+						}
+						&.expanded.parent ul {
+							padding-left: 5px;
+							ul {
+								padding-left: 0;
+								li {
+									padding-left: 0;
+									ul {
+										padding-left: 5px;
+										ul {
+											padding-left: 0;
+										}
+									}
 								}
 							}
 						}
 					}
 				}
 			}
 		}
-	}
-
-	canjs-sidebar {
-		button {
-			background: transparent;
-			border: 0;
-			color: @link-color;
-			cursor: pointer;
-			padding: 0;
-			position: absolute;
-			right: 0 - @gutter;
-			top: 0;
-		}
-		button > span {
-			font-family: monospace;// makes the + / - buttons the same width
-			font-size: 130%;
-			padding: (@gutter * .25) @gutter (@gutter * .6);
-		}
-		button:focus, button > span:focus {
-			outline: 0;
+		@media screen and (max-width: @breakpoint) {
+			& > ul > li.expanded.parent > ul {
+				padding-left: 5px;
+				// targets just the top level's subgroup to indent on mobile
+			}
 		}
-		button:focus > span {
-			border-radius: 5px;
-			box-shadow: 0 0 5px @link-color;
+		a.page.recipes + ul {
+			padding-left: 5px;
```

Comment:
I [removed the Recipes page](https://github.com/canjs/canjs/pull/4869) and updated the docMap on this branch, so I think this can probably be removed now.